db_access.py

```diff
@@ -27,6 +27,27 @@ def get_all_users():
     return conn.execute('SELECT id, handle, admin FROM user').fetchall()
 
 
+def user_for(uid):
+    conn = get_db()
+    for a in conn.execute('SELECT handle FROM user WHERE id=:id', {'id':uid}):
+        return a[0]
+    return ''
+
+
+def getUserLike(like_handle):
```

Comment:
This should be called `get_users_like` or `search_users_by_handle` -- note snake case for Python functions, and plural is useful for having a good idea what the function does.